src/org/thoughtcrime/securesms/recipients/Recipient.java

```diff
@@ -468,6 +477,14 @@ public void setRingtone(@Nullable Uri ringtone) {
     notifyListeners();
   }
 
+  public void setNickname(String nickname) {
+    synchronized (this) {
```

Comment:
is there a reason synchronized is not on the method (like for the others)

Reply:
That's the convention for the setters, check the rest of the file.

Reply:
:+1: